code/modules/materials/Mat_MaterialProcs.dm

```diff
@@ -318,19 +318,24 @@ triggerOnEntered(var/atom/owner, var/atom/entering)
 		if(prob(50) && owner && isturf(owner) && !isrestrictedz(T.z))
 			. = get_offset_target_turf(get_turf(entering), rand(-2, 2), rand(-2, 2))
 			entering.visible_message("<span class='alert'>[entering] is warped away!</span>")
-			boutput(entering, "<span class='alert'>You suddenly teleport ...</span>")
+			boutput(entering, "<span class='alert'>You suddenly teleport...</span>")
 			entering.set_loc(.)
 		return
 
 
 /datum/materialProc/telecrystal_onattack
 	execute(var/obj/item/owner, var/mob/attacker, var/mob/attacked)
 		var/turf/T = get_turf(attacked)
-		if(prob(50))
+		if(prob(33))
 			if(istype(attacked) && !isrestrictedz(T.z)) // Haine fix for undefined proc or verb /turf/simulated/floor/set loc()
 				. = get_offset_target_turf(get_turf(attacked), rand(-8, 8), rand(-8, 8))
+				var/fail_msg = ""
+				if (prob(25))
+					fail_msg = " but you lose [owner]!"
+					attacker.drop_item(owner)
+					playsound(attacker.loc, "sound/impact_sounds/Flesh_Tear_3.ogg", 40)
```

Comment:
reconsider the sound effect perhaps?

Reply:
Changed to "Poof". Also added sounds for the other warps which were mysteriously silent, will help make it more obvious "Oh I got warped".